### PR TITLE
create startup offer page

### PR DESCRIPTION
**Pull request by shughes26**
## Description
Page created based on [Figma file](https://www.figma.com/file/JTKHMxkmz8Isoql1oWEPhM/Draft-Startup-Offer?type=design&node-id=0-1&t=A9PahyZ0hD0dESR0-0#459531029) + feedback

Original Github issue [here](https://github.com/pulumi/pulumi-hugo/issues/2867)

## Checklist:
- [x] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [x] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [x] I have manually confirmed that all new links work.
- [x] I added aliases (i.e., redirects) for all filename changes.
- [x] If making css changes, I rebuilt the bundle.

### CODE REVIEW

**Comment by shughes26** [comment]
http://pulumi-hugo-origin-pr-2971-451b375e.s3-website.us-west-2.amazonaws.com/pulumi-for-startups/ page link on the preview

File: themes/default/layouts/page/pulumi-for-startups.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+{{ define "main" }}
+    <h2 class="mt-8 text-center">Get <b>$10,000</b> in Pulumi credits and exclusive help for your projects</h2>
+
+    <section id="success-stories" class="text-center p-8">
+        <div class="card mx-auto w-full lg:w-1/2 p-8">
+            <h3 class="text-gray-700 text-3xl font-medium">Pulumi Startup Success Stories</h3>
+            <div class="flex lg:flex-row flex-wrap justify-center space-x-4 spacy-y-4">
+                <a href="/case-studies/sourcegraph/"><img src="/logos/customers/sourcegraph-logo-cropped.svg" class="h-6 lg:h-8 my-2" alt="Sourcegraph" /></a>
+                <a href="/case-studies/panther-labs/"><img src="/logos/customers/panther-labs-wordmark.svg" class="h-6 lg:h-8 my-2" alt="Panther" /></a>
+                <img src="/logos/customers/cockroach-labs-logo.svg" class="h-6 lg:h-8 my-2" alt="CockroachDB" />
```
Comment (sean1588) [inline themes/default/layouts/page/pulumi-for-startups.html:22]: this seems to be missing the link to the cockroach case study

**Review by sean1588 — APPROVED** [review]
looks like the cockroach link is not working, but other than that looks good to me.

File: themes/default/layouts/page/pulumi-for-startups.html (hunk 1 above)
Comment (shughes26) [reply themes/default/layouts/page/pulumi-for-startups.html:22]: Yeah it's a little odd to have one missing link, but we don't currently have a cockroach case study to link to

**Pull request merged**